### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by fabjab86**

### CODE REVIEW

**Review by Eviekim — APPROVED** [review]

**Review by ollie28 — APPROVED** [review]
I approve!

File: FEMO/Space_board/GameScene.swift [hunk 1]
```diff
@@ -52,6 +52,11 @@ class GameScene: SKScene {
     override func didMove(to view: SKView) {
         setupTiles()
         createPlayer1()
+        let menu_buttonNode = self.childNode(withName: "Menu_button") as! SKSpriteNode
```
Comment (Mschmidt19) [inline FEMO/Space_board/GameScene.swift:57]: We may want to declare the button at the top of the file as we did with the other nodes

File: FEMO/Space_board/GameScene.swift [hunk 2]
```diff
@@ -102,12 +107,21 @@ class GameScene: SKScene {
...
-            let node = self.nodes(at: location).first
+        let touch = touches.first
+        
+        if let location = touch?.location(in: self) {
+            let node = self.nodes(at: location)
             
-            if node?.name == "nextTileButton" {
+            if node.first?.name == "nextTileButton" {
                 playTurn()
+            } else if node.first?.name == "Menu_button" {
```
Comment (Mschmidt19) [inline FEMO/Space_board/GameScene.swift:119]: Looks great! We just need to add a line here to save the state of the board when transitioning. Same thing goes for the info button.

**Review by Mschmidt19 — APPROVED** [review]

File: FEMO/Space_board/GameScene.swift (hunk 1 above)
Comment (fabjab86) [reply FEMO/Space_board/GameScene.swift:57]: completed

File: FEMO/Space_board/GameScene.swift (hunk 2 above)
Comment (fabjab86) [reply FEMO/Space_board/GameScene.swift:119]: completed

**Pull request merged**